### PR TITLE
Entire development of NYT clone

### PR DESCRIPTION
**Pull request by martinnajjar12**
### NYT Clone
We've used the techs below to create this website:

- HTML5
- CSS3
- Linters

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,595 @@
...
+.left {
+  color: black;
+}
+
+.left * {
+  margin: 0 5px;
+}
+
+.left-div a {
+  text-decoration: none;
```
Comment (kevcap) [inline styles.css]: - [ ] Remove all the unnecessarily repeated CSS code. Keeping your code DRY is an excellent practice that will help you a lot and make your code more professional. (Check this link form better understanding [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
For example: Group all the rules that are common to all the anchor tags:
image

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,595 @@
...
+.video-container {
+  width: 100%;
+  height: 0;
+  padding-bottom: 56.25%;
+  position: relative;
+  z-index: 10;
+}
+
+.video-container iframe {
+  width: 100%;
```
Comment (kevcap) [inline styles.css]: - [ ] Remove all the unnecessarily repeated CSS code. Keeping your code DRY is an excellent practice that will help you a lot and make your code more professional. (Check this link form better understanding [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,595 @@
...
+  margin-right: 15px;
+}
+
+.bottom-img-div {
+  width: 272px;
+  height: 181px;
+}
+
+.flex-item img {
+  width: 100%;
```
Comment (kevcap) [inline styles.css]: - [ ] Remove all the unnecessarily repeated CSS code. Keeping your code DRY is an excellent practice that will help you a lot and make your code more professional. (Check this link form better understanding [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,595 @@
...
+
+.right-div h2:hover {
+  text-decoration: underline;
+}
+
+/* ----------------------- BOTTOM NAVBAR --------------------- */
+
+.bottom-nav {
+  display: inline-block;
+  width: 100%;
```
Comment (kevcap) [inline styles.css]: - [ ] Remove all the unnecessarily repeated CSS code. Keeping your code DRY is an excellent practice that will help you a lot and make your code more professional. (Check this link form better understanding [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,595 @@
...
+}
+
+.main-image {
+  max-width: 1200px;
+  width: 100%;
+  margin: auto;
+}
+
+.main-image img {
+  width: 100%;
```
Comment (kevcap) [inline styles.css]: - [ ] Remove all the unnecessarily repeated CSS code. Keeping your code DRY is an excellent practice that will help you a lot and make your code more professional. (Check this link form better understanding [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,595 @@
...
+}
+
+.aside-title {
+  border-bottom: 1px solid #eee;
+  padding-bottom: 10px;
+  margin-bottom: 10px;
+}
+
+.picks article {
+  width: 100%;
```
Comment (kevcap) [inline styles.css]: - [ ] Remove all the unnecessarily repeated CSS code. Keeping your code DRY is an excellent practice that will help you a lot and make your code more professional. (Check this link form better understanding [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)

File: styles.css [hunk 7]
```diff
@@ -0,0 +1,595 @@
...
+/* -------------------- THE SECOND MAIN ----------------------- */
+
+.second-main {
+  width: 100%;
+  margin: 0 auto;
+}
+
+.container-article {
+  max-width: 1000px;
+  width: 100%;
```
Comment (kevcap) [inline styles.css]: - [ ] Remove all the unnecessarily repeated CSS code. Keeping your code DRY is an excellent practice that will help you a lot and make your code more professional. (Check this link form better understanding [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)

**Review by kevcap — CHANGES_REQUESTED** [review]
# [STATUS] CHANGES REQUIRED♻️

  ### Hi team, you guys did a very good job so far!
  #### I can see that you guys are working hard at this project, and you're almost done, there are just some required changes* that you guys have to implement in order to proceed:

  _Note_: That changes marked as [OPTIONAL] are not required to approve your project, but I strongly recommend that you guys go for that extra mile, in order to get more knowledge and have a better project to add at your portfolio.
  
## [REQUIRED CHANGES]
- [x] Guys, it should have a "Hamburger button", please implement that with a `<button>` tag (link to page) ([screenshot ](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/positioning_and_floating_requirements.png)from Pathwright)
- [x] Remember to make a descriptive README.md ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59882084/))
- [x] Files not related to the project should not be committed and added to .gitignore (e.g. package.json and package-lock.json) ([screenshot ](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/gitignore.png)from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
- [x] It should have a `<footer>` with fixed  position (it sticks to the bottom part of the screen) ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)) ([screenshot ](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/positioning_and_floating_requirements.png)from Pathwright)**
- [x] The NY Times logo should be a link ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)) ([screenshot ](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/positioning_and_floating_requirements.png)from Pathwright)
- [x] Kindly remove all the unnecessarily repeated CSS code. Keeping your code DRY is an excellent practice that will help you a lot and make your code more professional. (Check this link form better understanding [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
Take a look at this example:
image
Your HTML will be something like this: 
image
Apply the same logic above in **ALL** those repeated rules listed below:
image
- [x] Please add a live demo link. (You can use [Raw Githack](https://raw.githack.com/), for example.)



## [OPTIONAL CHANGES]
- [ ] Put your `css` files in a css folder. (To make your repo more professional)

## Summary
✨ Good job with that project, it looks really good for a first review 💥
✨ Well done implementing `flex`, `grid` and `float` ! The layout looks almost the same.
## Please make the required changes and resubmit for a new code review once you are ready💪

## As always, I wish you all the very best and see you both at the top🚀

  👨🏻‍💻 **TSE: Kevin Alves**

    

  📫 Have any question? Better call Kevin:

  <a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

<sub>*We're here to help you, to ensure that you're using the best practices and getting you ready for the job market. In real-world, you'll have a lot of changes requests for your real clients, it's good to get used to it.</sub>
<sub>**Some Students aren't being able to see the sticky footer, use [this](https://user-images.githubusercontent.com/56762529/90828132-dd9a9280-e313-11ea-8a52-9d4cbefbedc6.png) image as a guide
 </sub>

**Comment by martinnajjar12** [comment]
> # [STATUS] CHANGES REQUIRED♻️
> ### Hi team, you guys did a very good job so far!
> #### I can see that you guys are working hard at this project, and you're almost done, there are just some required changes* that you guys have to implement in order to proceed:
> _Note_: That changes marked as [OPTIONAL] are not required to approve your project, but I strongly recommend that you guys go for that extra mile, in order to get more knowledge and have a better project to add at your portfolio.
> 
> ## [REQUIRED CHANGES]
> * [x]  Guys, it should have a "Hamburger button", please implement that with a `<button>` tag (link to page) ([screenshot ](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/positioning_and_floating_requirements.png)from Pathwright)
> * [x]  Remember to make a descriptive README.md ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59882084/))
> * [x]  Files not related to the project should not be committed and added to .gitignore (e.g. package.json and package-lock.json) ([screenshot ](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/gitignore.png)from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
> * [x]  It should have a `<footer>` with fixed  position (it sticks to the bottom part of the screen) ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)) ([screenshot ](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/positioning_and_floating_requirements.png)from Pathwright)**
> * [x]  The NY Times logo should be a link ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)) ([screenshot ](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/positioning_and_floating_requirements.png)from Pathwright)
> * [x]  Kindly remove all the unnecessarily repeated CSS code. Keeping your code DRY is an excellent practice that will help you a lot and make your code more professional. (Check this link form better understanding [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
>   Take a look at this example:
>   image
>   Your HTML will be something like this:
>   image
>   Apply the same logic above in **ALL** those repeated rules listed below:
>   image
> * [x]  Please add a live demo link. (You can use [Raw Githack](https://raw.githack.com/), for example.)
> 
> ## [OPTIONAL CHANGES]
> * [ ]  Put your `css` files in a css folder. (To make your repo more professional)
> 
> ## Summary
> ✨ Good job with that project, it looks really good for a first review 💥
> ✨ Well done implementing `flex`, `grid` and `float` ! The layout looks almost the same.
> 
> ## Please make the required changes and resubmit for a new code review once you are ready💪
> ## As always, I wish you all the very best and see you both at the top🚀
> 👨🏻‍💻 **TSE: Kevin Alves**
> 
> 📫 Have any question? Better call Kevin:
> 
> [ gmail](mailto:kevinalves.js@gmail.com) [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)
> 
> *We're here to help you, to ensure that you're using the best practices and getting you ready for the job market. In real-world, you'll have a lot of changes requests for your real clients, it's good to get used to it.
> **Some Students aren't being able to see the sticky footer, use [this](https://user-images.githubusercontent.com/56762529/90828132-dd9a9280-e313-11ea-8a52-9d4cbefbedc6.png) image as a guide

Thank you for your review, We really appreciate it :)

**Review by SpaYco — CHANGES_REQUESTED** [review]
# Changes Requested 

Hello Team! 👋 

Awesome job so far!! 👍 good job implementing the changes, you forgot one though! 

## Required Changes ♻️ 

- [x]  Files not related to the project should not be committed and added to .gitignore (e.g. package.json and package-lock.json) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/gitignore.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)) (Hint: Use `git rm file_name`)

### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Comment by SpaYco** [comment]
### Please note, if you wanna remove the files from GitHub, but keep it locally, use .gitignore then [follow this](https://stackoverflow.com/questions/3469741/remove-file-from-the-repository-but-keep-it-locally)

**Review by SpaYco — APPROVED** [review]
# Project Approved ✔️ 

thank you for making the required changes, your project is now approved!

### please don't forget to merge to master, good luck with the next projects!


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Pull request merged**